llvm/lib/Target/X86/X86AsmPrinter.cpp

```diff
@@ -975,6 +976,28 @@ static void emitNonLazyStubs(MachineModuleInfo *MMI, MCStreamer &OutStreamer) {
   }
 }
 
+/// True if this module is being built for windows/msvc, and uses floating
+/// point.  This is used to emit an undefined reference to _fltused.
```

Comment:
Please add a FIXME or TODO about computing this from MIR instead of the IR, and include some context about how this is used in Windows kernel or driver contexts to find and prevent code from modifying non-GPR registers.